Limitations.md

```diff
@@ -0,0 +1,315 @@
+
+## Networking limitations
+
+### Support for joining an existing VM network
+
+Docker supports the ability for containers to join another containers
+namespace with the `docker run --net=containers` syntax. This allows
+multiple containers to share a common network namespace and the network
+interfaces placed in the network namespace.  Kata Containers does not
+support network namespace sharing. If a Kata Container is setup to
+share the network namespace of a `runc` container, the runtime
+effectively takes over all the network interfaces assigned to the
+namespace and binds them to the VM. Consequently, the `runc` container loses
+its network connectivity.
+
+### docker --net=host
+
+Docker host network support (`docker --net=host run`) is not supported.
+It is not possible to directly access the host networking configuration
+from within the VM.
+
+The `--net=host` option can still be used with `runc` containers and
+inter-mixed with running Kata Containers, thus enabling use of `--net=host`
+when necessary.
+
+It should be noted, currently passing the `--net=host` option into a
+Kata Container may result in the Kata Container networking setup
+modifying, re-configuring and therefore possibly breaking the host
+networking setup. Do not use `--net=host` with Kata Containers.
+
+### docker run --link
+
+The runtime does not support the `docker run --link` command. This
+command is now deprecated by docker and we have no intention of adding support.
+Equivalent functionality can be achieved with the newer docker networking commands.
+
+See more documentation at
+[docs.docker.com](https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/).
+
+## Host resource sharing
+
+### docker run --privileged
+
+The `docker run --privileged` command is not supported in the runtime.
+There is no simple way to grant the VM access to all of the host devices that this command needs to be complete.
+
+The `--privileged` option can be used with `runc` containers and inter-mixed with running Kata Containers. This enables use of `--privileged` when necessary.
+
+# Miscellaneous
+
+This section lists limitations where the possible solutions are uncertain.
+
+## Docker ramdisk not supported
+
+The magic `DOCKER_RAMDISK=true` environment variable used by Docker to force
+the container to run entirely on a RAM disk is not supported.
+
+See https://github.com/kata-containers/runtime/issues/134 for more information.
+
+# Appendices
+
+## The constraints challenge
+
+Applying resource constraints such as cgroup, cpu, memory, and storage to a workload is not always straightforward with a VM based system. A Kata Container runs in an isolated environment inside a virtual machine. This, coupled with the architecture of Kata Containers, offers many more possibilities than are available to traditional Linux* containers due to the various layers and contexts.
```

Comment:
'Linux' is mentioned previously on line 222 - you might want to move the `*` up there as the 'first mention'.

Reply:
Done.